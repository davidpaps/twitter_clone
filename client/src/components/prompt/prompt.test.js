import React from "react";
import { shallow } from "enzyme";
import Prompt from "./prompt";
import renderer from "react-test-renderer";

describe("Prompt", () => {
  let wrapper;

  beforeEach(() => (wrapper = shallow(<Prompt />)));

  it("should render a <Fragment />", () => {
    expect(wrapper.find("Fragment").length).toEqual(1);
  });

  it("should render correctly", () => {
    const tree = renderer.create(<Prompt />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
