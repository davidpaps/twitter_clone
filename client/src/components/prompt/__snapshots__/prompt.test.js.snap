// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prompt should render correctly 1`] = `null`;
